### ISSUE TITLE
[1.19] Forge crashes when trying to use this mod

### ISSUE THREAD

**Issue by elamperti** [North-West-Wind/forge-autofish#16]
**Describe the bug**
The mod makes Forge crash on Minecraft 1.19

**To Reproduce**
- Install Forge for 1.19
- Add this mod (`forgeautofish-5.0.1-1.19.jar`, downloaded from CurseForge) to the mods directory
- Try to start forge

**Expected behavior**
Minecraft should start and the mod should work.

**Screenshots**
![Selection_001](https://user-images.githubusercontent.com/910672/178128474-e3736096-9c7c-4c9b-879f-33abd7f2ece3.png)

**Crash report** (fragment)
```
	Mod List: 
	client-1.19-20220607.102129-srg.jar               |Minecraft                     |minecraft                     |1.19                |COMMON_SET|Manifest: (...)
	forge-1.19-41.0.87-universal.jar                  |Forge                         |forge                         |41.0.87             |COMMON_SET|Manifest: (...)
 -->	forgeautofish-5.0.1-1.19.jar                      |AutoFish for Forge            |forgeautofish                 |5.0.1               |ERROR     |Manifest: NOSIGNATURE
	InventoryProfilesNext-forge-1.19-1.5.3.jar        |Inventory Profiles Next       |inventoryprofilesnext         |1.5.3               |COMMON_SET|Manifest: NOSIGNATURE
	appleskin-forge-mc1.19-2.4.0.jar                  |AppleSkin                     |appleskin                     |2.4.0+mc1.19        |COMMON_SET|Manifest: NOSIGNATURE
	journeymap-1.19-5.8.5rc3-forge.jar                |Journeymap                    |journeymap                    |5.8.5rc3            |COMMON_SET|Manifest: NOSIGNATURE
```

**Comment by elamperti** [comment]
Despite what the crash report suggests, this may be related to [these breaking changes](https://github.com/MinecraftForge/MinecraftForge/pull/8786) in Forge, as Forge versions 1.19.064 and above crash with the same error even after removing this mod and leaving others. I'm unfamiliar with the underlying changes and if there's work needed on this mods' side, so feel free to close this issue if the problem is upstream.

**Comment by North-West-Wind** [comment]
Fixed with v5.0.2
